apps/hubble-stats/containers/OverviewChipsContainer/OverviewChipsContainer.tsx

```diff
@@ -11,12 +11,18 @@ export default async function OverviewChipsContainer() {
     <div className="hidden md:flex items-center gap-4">
       <Chip color="blue">
         <BlockIcon size="lg" className="stroke-blue-90 dark:stroke-blue-30" />
-        TVL: ${numbro(tvl).format({ thousandSeparated: true })}
+        TVL: $
+        {tvl
```

Comment:
You can use `typeof tvl` to verify the type. We continue to display the `tvl` even when `tvl = 0`. If you follow this approach, `0` will be interpreted as `false`.